### PR TITLE
Succulent wolverine - some bug fixes

### PR DESCRIPTION
**Pull request by clottman**
https://succulent-wolverine.glitch.me/ 
https://glitch.com/~succulent-wolverine

- .babelrc changes: makes sure we get some polyfills for the browsers that error the most
  - Eventually it would be nice if we can load these conditionally based on browser/feature detects. For the short term I think making things work should be a priority over bundle size. 
- DefaultAvatar: clean this component up; hopefully avoid a timing issue we seemed to have had where the `ref` was still valid, but the SVG wasn't there. 
  - Plus: removes a package from our bundle! (react-svg-inline)
- collection.js: That prop is not actually required
- login.js: some browsers don't support window.origin; since we always know what window.origin is supposed to be we can just hard code it.

### CODE REVIEW

File: src/components/collection/defaultAvatar.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        stroke="#000"
+        strokeWidth={4}
+        fill="#FFF"
+      />
+    </g>
+  </svg>
+);
+
+DefaultAvatar.defaultProps = {
+  backgroundFillColor: '#45C1F7',
```
Comment (GregWeil) [inline src/components/collection/defaultAvatar.js]: Does this need a proptypes, or is it implied by defaultprops?

File: src/presenters/includes/profile.js [hunk 2]
```diff
@@ -95,28 +82,24 @@ CoverContainer.defaultProps = {
...
-      </div>
-    </InfoContainer>
-    {!!teams && !!teams.length && (
-      <div className="teams-information">
-        <TeamsList teams={teams} />
-      </div>
-    )}
-  </CoverContainer>
-);
+export class ProfileContainer extends React.PureComponent {
```
Comment (clottman) [inline src/presenters/includes/profile.js]: Hoping to address too much recursion errors when a user uploads a new cover photo or profile picture. PureComponent does a shallow comparison instead of deep comparison

File: src/presenters/pages/login.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ function notifyParent(message = {}) {
   // Specifically target our same origin;
   // we're only communicating between the editor and its corresponding ~community site,
   // not across other environments.
-  const sameOrigin = window.origin;
+  const sameOrigin = 'https://glitch.com';
```
Comment (GregWeil) [inline src/presenters/pages/login.js]: This can use `APP_URL`

File: package.json [hunk 4]
```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "description": "Find cool projects and people using Glitch",
   "scripts": {
-    "prestart": "nodemon --watch server --watch shared --exec 'eslint --config server/.eslintrc.server.js server shared webpack.config.js' &",
-    "start": "nodemon --watch server --watch shared --watch views ./server/server.js",
+    "prestart": "node_modules/.bin/nodemon --watch server --watch shared --exec 'eslint --config server/.eslintrc.server.js server shared webpack.config.js' &",
+    "start": "node_modules/.bin/nodemon --watch server --watch shared --watch views ./server/server.js",
```
Comment (GregWeil) [inline package.json]: Why is this changed?

File: src/presenters/includes/profile.js [hunk 5]
```diff
@@ -9,22 +9,17 @@ import Button from '../../components/buttons/button';
...
   <div className="upload-image-buttons">
-    { !!uploadImage && (
+    {!!uploadImage && (
       <TrackClick name={`Upload ${name}`}>
-
         <Button size="small" type="tertiary" onClick={uploadImage}>
-            Upload
-          {' '}
-          {name}
+          Upload {name}
```
Comment (GregWeil) [inline src/presenters/includes/profile.js]: Putting two expressions on the same line breaks an eslint rule we have, but I'd be in favor of disabling it

File: package.json (hunk 4 above)
Comment (clottman) [reply package.json]: I assumed it was changed in ~community right before I remixed - I think one of the infrastructure team members tried it to fix node_modules. If we can safely change back, I'll do that.

File: src/presenters/includes/profile.js (hunk 5 above)
Comment (clottman) [reply src/presenters/includes/profile.js]: Me too - that one seems to cause more less-readable code than it's worth.

File: src/components/collection/defaultAvatar.js (hunk 1 above)
Comment (clottman) [reply src/components/collection/defaultAvatar.js]: Eslint didn't seem to hate it as-is, but I can add it in.
